### DISCUSSION TITLE
How to make foreground element transparent?

### DISCUSSION THREAD

**Discussion by deg0nz** [starship/starship#4896] [Q&A]
Hi,

I'm having trouble making the foreground of a unicode element transparent. I want to achieve an effect like the divider in the `oh-my-posh` default theme: ![](https://ohmyposh.dev/img/hero.png)

This can be achieved by setting the foregroung of the `` sign (Unicode `U+E0B0`) to `transparent`.

When I set `fg` of the sign to `transparent`, it stays white. Setting `bg` to `transparent` seems to work.

I'm using iTerm2 as terminal emulator and understand, that there is an issue with iTerm and `hidden` as mentioned in the [`starship` docs](https://starship.rs/advanced-config/#style-strings).  Is it related to `transparent`?

The thing is: When trying `oh-my-posh` in the same terminal, the effect is working. They set their transparent [like this](https://github.com/JanDeDobbeleer/oh-my-posh/blob/195c1a934d66fd56f757b1235cc957e244b1c3bd/src/ansi/ansi_writer.go#L58).

Am I missing something in the Starship config/docs, or is it just not working?

**Comment by a-usr** [comment]
I cant get it to work either, and I am using windows terminal. So yeah..

**Comment by a-usr** [comment]
For the time being you could set the color you want as your background to be the foreground color, and then specify "inverted" afterwards
